haunted-houses/src/main/java/cz/muni/fi/pa165/hauntedhouses/dao/AbilityDao.java

```diff
@@ -1,4 +1,24 @@
 package cz.muni.fi.pa165.hauntedhouses.dao;
 
+import cz.muni.fi.pa165.hauntedhouses.model.Ability;
+
+import java.util.List;
+
+/**
+ * @author David Hofman
+ */
 public interface AbilityDao {
+
+    void createAbility(Ability A);
+
+    Ability readAbilityId(Long id);
+
+    Ability readAbilityName(String name);
```

Comment:
Tady jsme se shodli, že "read" úplně nevystihuje, co ta funkce dělá. Místo toho používáme "getEntityByParam", takže např. tady by to bylo "getAbilityByName".

Reply:
Zkusil jsem git rebase, protože to mám ve dvou commitech a amend funguje jen na ten poslední. Vypadá to, že se to povedlo...